src/main/java/org/radarbase/appserver/controller/FcmNotificationController.java

```diff
@@ -262,7 +262,9 @@ public FcmNotificationController(FcmNotificationService notificationService) {
           + "/"
           + PathsUtil.SUBJECT_ID_CONSTANT
           + "/"
-          + PathsUtil.MESSAGING_NOTIFICATION_PATH)
+          + PathsUtil.MESSAGING_NOTIFICATION_PATH
+          + "/"
+          + PathsUtil.ALL_KEYWORD)
```

Comment:
also similar for Data messages?

Reply:
> Please find a comment.
> 
> Also, will this be a breaking change for current app versions? I assume not but If so, we should mention this in the release.

No since we currently don't use the path for deleting all notifications. We only use the path for deleting single notifications.